packages/api/app/Controllers/Http/KnowledgeAreaController.js

```diff
@@ -10,9 +11,14 @@ class KnowledgeAreaController {
 
 	async index({ request }) {
 		const filters = request.all();
-		return KnowledgeArea.query()
-			.withFilters(filters)
-			.fetch();
+		const key = cache.generateKey(KnowledgeArea.name, filters);
+		const oneDayInSeconds = 60 * 60 * 24;
+
+		return cache.remember(key, oneDayInSeconds, async () => {
```

Comment:
Resolve the cached response or process the block of code in brackets then cache it